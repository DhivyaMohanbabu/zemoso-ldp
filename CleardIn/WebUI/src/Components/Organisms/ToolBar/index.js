/* eslint-disable arrow-body-style */
/* eslint-disable react/forbid-prop-types */
import React from 'react';
import { Grid, makeStyles, TextField } from '@material-ui/core';
import PropTypes from 'prop-types';
import SearchIcon from '@material-ui/icons/Search';
import ViewStreamOutlinedIcon from '@material-ui/icons/ViewStreamOutlined';
import ActionOption from '../../Molecules/ActionOption';
import DropDown from '../DropDown';

const useStyles = makeStyles((theme) => ({
  root: {
    padding: '5px 10px',
    backgroundColor: '#ffff',
    boxShadow: '0 4px 10px 0 rgba(25, 1, 52, 0.16)',
    border: '1px',
    borderColor: theme.palette.info.main,
    height: '54px',
    color: theme.palette.clearedIn.grey[500],
  },
  groupBox: {
    alignSelf: 'center',
  },
  dropDownBox: {
    paddingRight: '16px',
  },
  input: {
    borderRadius: '100px',
    height: '20px',
    padding: '4px',
    fontSize: '16px',
    width: '130px',
  },
  searchBarBox: {
    padding: '8px 0px',
  },
  actionBox: {
    height: '24px',
    paddingRight: '15px',
  },
}));
const ToolBar = ({
  userActions,
  placeholder,
}) => {
  const classes = useStyles();
  const [display, setDisplay] = React.useState(false);
  const [value, setValue] = React.useState('');
  const handleClick = () => {
    setDisplay(!display);
  };

  const handleChange = (e) => {
    setValue(e.target.value);
  };
  const isSecurityDropDown = false;
  const names = [{ label: 'Sales', icon: ViewStreamOutlinedIcon }, { label: 'Department', icon: ViewStreamOutlinedIcon }];
  console.log(userActions);

  return (
    <Grid container className={classes.root} justify="space-between" data-testid="toolbar">
      <Grid item className={classes.groupBox}>
        <Grid container alignItems="center">
          <Grid item className={classes.dropDownBox}>
            <div style={{ display: 'flex', borderRight: '1px solid grey' }} data-testid="toolbar1">
              <DropDown
                isSecurityDropDown={isSecurityDropDown}
                names={names}
                value="Sales"
              />
            </div>
          </Grid>
          <Grid item>
            <Grid container alignItems="center">
              {userActions && userActions.map((action, index) => {
                return (
                  <Grid item key={index} className={classes.actionBox}>
                    <ActionOption icon={action.icon} label={action.label} count={action.count} />
                  </Grid>
                );
              })}
              {/*
                <Grid item key={index} className={classes.actionBox}>
                  <ActionOption icon={action.icon} label={action.label} count={action.count} />
                </Grid>
              ))} */}
            </Grid>
          </Grid>
        </Grid>
      </Grid>

      <Grid item>
        <Grid container className={classes.searchBarBox}>
          <Grid item onClick={handleClick} data-testid="handleClick">
            <SearchIcon />
          </Grid>
          <Grid item>
            {display && (
            <TextField aria-label="text" placeholder={placeholder} value={value} variant="outlined" onChange={(e) => handleChange(e)} className={classes.textField} inputProps={{ className: classes.input }} />
            )}
          </Grid>
        </Grid>
      </Grid>
    </Grid>
  );
};

ToolBar.propTypes = {
  userActions: PropTypes.arrayOf(Object).isRequired,
  placeholder: PropTypes.string.isRequired,
};
export default ToolBar;
