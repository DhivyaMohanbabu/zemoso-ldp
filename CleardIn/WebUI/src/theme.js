import { grey } from '@material-ui/core/colors';
import { createMuiTheme } from '@material-ui/core/styles';
// import 'typeface-roboto';

const theme = createMuiTheme({
  palette: {
    primary: {
      main: '#1665d8',
      light: '#b0f1f0',
    },
    secondary: {
      main: '#9ea0a5',
      dark: grey,
    },
    error: {
      main: '#ffada5',

    },
    success: {
      main: '#daefa9',
    },
    info: {
      main: grey[900],
      light: '#fdd570',
      dark: grey[600],
    },
    text: {
      primary: '#1c1c21', // '#3e3f42',
      secondary: '#5c5c5c', // #6b6c6f',
    },
    grey: {
      100: '#f8fafb',
      200: '#f2f4f6',
      300: '#e8ecef',
      400: '#98a9bc',
      500: '#778ca2',
    },
    clearedIn: {
      grey: {
        100: '#f6f6f6',
        200: '#eeeeee',
        300: '#cccccc',
        400: '#949494',
        500: '#666666',
      },
      selectedElementBGColor: '#eaf0ff', // rgba(77, 124, 254, 0.12),
      dark: '#5c5c5c',
      bodyBackground: '#fafafa',
      blue: '#1d4cd7',
    },
    relationShip: {
      vendor: '#daefa9',
      customer: '#fdd570',
      partner: '#b0f1f0',
    },
  },
  typography: {
    fontFamily: ['Roboto', 'Rubik', 'sans-serif'].join(','),
    h1: {
      fontSize: '1.875rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    h2: {
      fontSize: '1.625rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    h3: {
      fontSize: '1.25rem',
      fontWeight: 300,
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    h4: {
      fontSize: '1.125rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    h5: {
      fontSize: '1rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 1.5,
      letterSpacing: 'normal',
    },
    h6: {
      fontSize: '1.25rem',
      fontWeight: 500,
      fontStyle: 'normal',
      lineHeight: 1.05,
      letterSpacing: 'normal',
    },
    subtitle1: {
      fontSize: '0.938rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    subtitle2: {
      fontSize: '0.938rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    caption: {
      fontSize: '0.875rem',
      fontWeight: 500,
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
    },
    link: {
      fontSize: '0.875rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 1.5,
      letterSpacing: 'normal',
    },
    body1: {
      fontSize: '0.875rem',
      fontWeight: 'normal',
      fontStyle: 'normal',
      lineHeight: 1.14,
      letterSpacing: 'normal',
    },
    body2: {
      fontFamily: 'Rubik',
      fontSize: '0.875rem',
      fontWeight: 300,
      fontStyle: 'normal',
      lineHeight: 1.5,
      letterSpacing: 'normal',
    },
    button: {
      fontSize: '0.75rem',
      fontWeight: 500,
      fontStyle: 'normal',
      lineHeight: 'normal',
      letterSpacing: 'normal',
      textTransform: 'none',
    },
  },
  overrides: {
    MuiButton: {
      root: {
        maxWidth: '200px',
        display: 'block',
      },
      label: {
        whiteSpace: 'nowrap',
        overflow: 'hidden',
        textOverflow: 'ellipsis',
      },
    },
  },
});

export default theme;
