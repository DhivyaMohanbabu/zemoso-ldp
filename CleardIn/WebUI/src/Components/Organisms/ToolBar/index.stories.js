import React from 'react';
import MailOutlineOutlinedIcon from '@material-ui/icons/MailOutlineOutlined';
import FilterListOutlinedIcon from '@material-ui/icons/FilterListOutlined';
import ListIcon from '@material-ui/icons/List';
import ImportExportOutlinedIcon from '@material-ui/icons/ImportExportOutlined';
import DeleteForeverOutlinedIcon from '@material-ui/icons/DeleteForeverOutlined';
import ToolBar from '.';

export default {
  title: 'Organism/ToolBar',
};

const Template = (args) => <ToolBar {...args} />;

export const Default = Template.bind({});
Default.args = {
  placeholder: 'Find a Domain',

  dropDownOptions: [
    {
      label: 'Sales',
      icon: MailOutlineOutlinedIcon,
    },
    {
      label: 'Legal',
      icon: MailOutlineOutlinedIcon,
    },
    {
      label: 'Finance',
      icon: MailOutlineOutlinedIcon,
    },
    {
      label: 'Investor relations',
      icon: MailOutlineOutlinedIcon,
    },
  ],
  userActions: [{
    icon: FilterListOutlinedIcon,
    label: 'Filter',
  },
  {
    icon: ListIcon,
    label: 'Group',
  },
  {
    icon: ImportExportOutlinedIcon,
    label: 'Sort',
  },
  {
    icon: DeleteForeverOutlinedIcon,
    label: 'Delete Selected',
    count: 3,
  },
  ],
};
