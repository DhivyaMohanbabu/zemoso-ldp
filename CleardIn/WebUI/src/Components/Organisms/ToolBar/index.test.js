/* eslint-disable no-unused-expressions */
import React from 'react';
import { fireEvent, render, screen } from '@testing-library/react';
import { ThemeProvider } from '@material-ui/core/styles';
import FilterListOutlinedIcon from '@material-ui/icons/FilterListOutlined';
import ListIcon from '@material-ui/icons/List';
import ImportExportOutlinedIcon from '@material-ui/icons/ImportExportOutlined';
import DeleteForeverOutlinedIcon from '@material-ui/icons/DeleteForeverOutlined';
import theme from '../../../theme';
import ToolBar from '.';

import '@testing-library/jest-dom';

describe('ToolBar', () => {
  it('should render ToolBar ', () => {
    const userActions = [{
      icon: FilterListOutlinedIcon,
      label: 'Filter',
    },
    {
      icon: ListIcon,
      label: 'Group',
    },
    {
      icon: ImportExportOutlinedIcon,
      label: 'Sort',
    },
    {
      icon: DeleteForeverOutlinedIcon,
      label: 'Delete Selected',
      count: 3,
    },
    ];
    const container = render(
      <ThemeProvider theme={theme}>
        <ToolBar placeholder="find an option" userActions={userActions} />
      </ThemeProvider>,
    );
    expect(container).toMatchSnapshot();
  });
  it('should render ToolBar Organism', () => {
    const userActions = [{
      icon: FilterListOutlinedIcon,
      label: 'Filter',
    },
    {
      icon: ListIcon,
      label: 'Group',
    },
    {
      icon: ImportExportOutlinedIcon,
      label: 'Sort',
    },
    {
      icon: DeleteForeverOutlinedIcon,
      label: 'Delete Selected',
      count: 3,
    },
    ];
    const { getByTestId, getByText, getByLabelText } = render(
      <ThemeProvider theme={theme}>
        <ToolBar placeholder="find a domain" userActions={userActions} />
      </ThemeProvider>,
    );
    getByTestId('toolbar');
    fireEvent.change(screen.getByTestId('toolbar'), {
      target: {
        icon: FilterListOutlinedIcon,
        label: 'Filter',
      },
    });
    const component = getByTestId('toolbar');

    expect(component).not.toBeNull();
    const component2 = getByTestId('handleClick');
    expect(component2).not.toBeNull();
    fireEvent.click(component2);
    fireEvent.click(getByText('Delete Selected'));
    expect(getByText('Delete Selected')).toBeVisible();
    const component3 = getByLabelText('text');
    fireEvent.change(component3, jest.fn());
  });
  it('should render Security Option ', () => {
    const mock = jest.fn();
    expect(mock.mock.calls).toEqual([]);
  });
});
