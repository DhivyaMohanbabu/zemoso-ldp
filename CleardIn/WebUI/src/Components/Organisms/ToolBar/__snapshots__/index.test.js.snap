// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolBar should render ToolBar  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-justify-xs-space-between"
        data-testid="toolbar"
      >
        <div
          class="MuiGrid-root makeStyles-groupBox-2 MuiGrid-item"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
          >
            <div
              class="MuiGrid-root makeStyles-dropDownBox-3 MuiGrid-item"
            >
              <div
                data-testid="toolbar1"
                style="display: flex; border-right: 1px solid grey;"
              >
                <div>
                  <div
                    class="MuiFormControl-root makeStyles-root-7"
                  >
                    <div
                      class="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
                      data-testid="select"
                    >
                      <div
                        aria-haspopup="listbox"
                        aria-labelledby="select-label demo-controlled-open-select"
                        class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
                        id="demo-controlled-open-select"
                        role="button"
                        tabindex="0"
                      >
                        <div
                          class="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
                        >
                          <div
                            class="MuiGrid-root MuiGrid-item"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-root makeStyles-icon-13 MuiSvgIcon-colorSecondary"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M4 6v12h17V6H4zm15 10H6v-3h13v3zM6 11V8h13v3H6z"
                              />
                            </svg>
                          </div>
                          <div
                            class="MuiGrid-root makeStyles-text-14 MuiGrid-item"
                          >
                            <p
                              class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap"
                            >
                              Sales
                            </p>
                          </div>
                        </div>
                      </div>
                      <input
                        aria-hidden="true"
                        class="MuiSelect-nativeInput"
                        tabindex="-1"
                        value="[object Object]"
                      />
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSelect-icon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
              >
                <div
                  class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                    display="inline"
                  >
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                    >
                      <p
                        class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                      >
                        Filter
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                    display="inline"
                  >
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M3 13h2v-2H3v2zm0 4h2v-2H3v2zm0-8h2V7H3v2zm4 4h14v-2H7v2zm0 4h14v-2H7v2zM7 7v2h14V7H7z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                    >
                      <p
                        class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                      >
                        Group
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                    display="inline"
                  >
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M9 3L5 6.99h3V14h2V6.99h3L9 3zm7 14.01V10h-2v7.01h-3L15 21l4-3.99h-3z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                    >
                      <p
                        class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                      >
                        Sort
                      </p>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                    display="inline"
                  >
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M14.12 10.47L12 12.59l-2.13-2.12-1.41 1.41L10.59 14l-2.12 2.12 1.41 1.41L12 15.41l2.12 2.12 1.41-1.41L13.41 14l2.12-2.12zM15.5 4l-1-1h-5l-1 1H5v2h14V4zM6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM8 9h8v10H8V9z"
                        />
                      </svg>
                    </div>
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                    >
                      <p
                        class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                      >
                        Delete Selected
                      </p>
                    </div>
                    <div
                      class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                      data-testid="count"
                    >
                      <p
                        class="MuiTypography-root makeStyles-text-15 MuiTypography-body1"
                      >
                        (
                        3
                        )
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiGrid-root makeStyles-searchBarBox-5 MuiGrid-container"
          >
            <div
              class="MuiGrid-root MuiGrid-item"
              data-testid="handleClick"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                />
              </svg>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-justify-xs-space-between"
      data-testid="toolbar"
    >
      <div
        class="MuiGrid-root makeStyles-groupBox-2 MuiGrid-item"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <div
            class="MuiGrid-root makeStyles-dropDownBox-3 MuiGrid-item"
          >
            <div
              data-testid="toolbar1"
              style="display: flex; border-right: 1px solid grey;"
            >
              <div>
                <div
                  class="MuiFormControl-root makeStyles-root-7"
                >
                  <div
                    class="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
                    data-testid="select"
                  >
                    <div
                      aria-haspopup="listbox"
                      aria-labelledby="select-label demo-controlled-open-select"
                      class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
                      id="demo-controlled-open-select"
                      role="button"
                      tabindex="0"
                    >
                      <div
                        class="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center"
                      >
                        <div
                          class="MuiGrid-root MuiGrid-item"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-root makeStyles-icon-13 MuiSvgIcon-colorSecondary"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M4 6v12h17V6H4zm15 10H6v-3h13v3zM6 11V8h13v3H6z"
                            />
                          </svg>
                        </div>
                        <div
                          class="MuiGrid-root makeStyles-text-14 MuiGrid-item"
                        >
                          <p
                            class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap"
                          >
                            Sales
                          </p>
                        </div>
                      </div>
                    </div>
                    <input
                      aria-hidden="true"
                      class="MuiSelect-nativeInput"
                      tabindex="-1"
                      value="[object Object]"
                    />
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSelect-icon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
            >
              <div
                class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
              >
                <div
                  class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                  display="inline"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                  >
                    <p
                      class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                    >
                      Filter
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
              >
                <div
                  class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                  display="inline"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M3 13h2v-2H3v2zm0 4h2v-2H3v2zm0-8h2V7H3v2zm4 4h14v-2H7v2zm0 4h14v-2H7v2zM7 7v2h14V7H7z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                  >
                    <p
                      class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                    >
                      Group
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
              >
                <div
                  class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                  display="inline"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M9 3L5 6.99h3V14h2V6.99h3L9 3zm7 14.01V10h-2v7.01h-3L15 21l4-3.99h-3z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                  >
                    <p
                      class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                    >
                      Sort
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root makeStyles-actionBox-6 MuiGrid-item"
              >
                <div
                  class="MuiGrid-root makeStyles-gridBox-16 MuiGrid-container"
                  display="inline"
                >
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 makeStyles-gridIcon-18 MuiGrid-item"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-root MuiSvgIcon-colorSecondary"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M14.12 10.47L12 12.59l-2.13-2.12-1.41 1.41L10.59 14l-2.12 2.12 1.41 1.41L12 15.41l2.12 2.12 1.41-1.41L13.41 14l2.12-2.12zM15.5 4l-1-1h-5l-1 1H5v2h14V4zM6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM8 9h8v10H8V9z"
                      />
                    </svg>
                  </div>
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                  >
                    <p
                      class="MuiTypography-root makeStyles-text-15 MuiTypography-body1 MuiTypography-noWrap"
                    >
                      Delete Selected
                    </p>
                  </div>
                  <div
                    class="MuiGrid-root makeStyles-gridBoxItem-17 MuiGrid-item"
                    data-testid="count"
                  >
                    <p
                      class="MuiTypography-root makeStyles-text-15 MuiTypography-body1"
                    >
                      (
                      3
                      )
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <div
          class="MuiGrid-root makeStyles-searchBarBox-5 MuiGrid-container"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
            data-testid="handleClick"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
